### PR TITLE
Vamr1 360 admin ui

### PR DESCRIPTION
**Pull request by danielstahr**
Finished UI for OpenVidu with new room API. 
Added Text-Chat.

### CODE REVIEW

**Comment by danielstahr** [comment]
Eigentlich war das der richtige UI Branch. Im Mute feature ging es nur um mute. Ich habe aber von diesem Branch gebranched, da ansonsten der Unterschied zum Master zu groß ist.

File: src/components/ChatComponent.vue [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  </div>
+</template>
+
+<script lang="ts">
+/* eslint-disable no-console */
+
+import { Component, Prop, Vue } from "vue-property-decorator";
+import { Session, SignalEvent } from "openvidu-browser";
+
+class Participant {
```
Comment (mithie) [inline src/components/ChatComponent.vue]: please also comment public classes and API for doc generation

File: src/components/ChatComponent.vue [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        ...this.participants,
+        {
+          id: name,
+          name: name
+          //imageUrl: "https://avatars3.githubusercontent.com/u/1915989?s=230&v=4"
+        }
+      ];
+    });
+  }
+  readonly agentProfile = {
```
Comment (mithie) [inline src/components/ChatComponent.vue]: shouldn't we make this configurable?

File: src/components/ChatComponent.vue [hunk 3]
```diff
@@ -0,0 +1,187 @@
+<template>
```
Comment (mithie) [inline src/components/ChatComponent.vue]: Is it common practice in vue to mix html with functional code? we should enforce strict separation of concerns. but maybe I got this wrong.

**Pull request closed without merging**